models/comments.model.js

```diff
@@ -16,7 +16,6 @@ exports.returnAllCommentsByArticleId = (article_id) => {
 }
 
 
-//nc-news-7
 exports.addNewCommentByArticleIdToDatabase = (article_id, username, body) => {
     article_id = Number(article_id)
```

Comment:
This is interesting...what would happen if I passed a string as the article_id? Would you expect an error once it's gone through this line?